ignite/cmd/model/testnet_multi_node.go

```diff
@@ -0,0 +1,162 @@
+package cmdmodel
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"syscall"
+
+	tea "github.com/charmbracelet/bubbletea"
+
+	"github.com/ignite/cli/v29/ignite/services/chain"
+)
+
+type NodeStatus int
+
+const (
+	Stopped NodeStatus = iota
+	Running
+)
+
+type Model struct {
+	appd string
+	args chain.MultiNodeArgs
+	ctx  context.Context
+
+	nodeStatuses []NodeStatus
+	pids         []int // Store the PIDs of the running processes
+	numNodes     int   // Number of nodes
+}
+
+type ToggleNodeMsg struct {
+	nodeIdx int
+}
+
+type UpdateStatusMsg struct {
+	nodeIdx int
+	status  NodeStatus
+}
+
+// Initialize the model
+func NewModel(chainname string, ctx context.Context, args chain.MultiNodeArgs) Model {
+	numNodes, err := strconv.Atoi(args.NumValidator)
+	if err != nil {
+		panic(err)
+	}
+	return Model{
+		appd:         chainname + "d",
+		args:         args,
+		ctx:          ctx,
+		nodeStatuses: make([]NodeStatus, numNodes), // initial states of nodes
+		pids:         make([]int, numNodes),
+		numNodes:     numNodes,
+	}
+}
+
+// Implement the Update function
+func (m Model) Init() tea.Cmd {
+	return nil
+}
+
+// ToggleNode toggles the state of a node
+func ToggleNode(nodeIdx int) tea.Cmd {
+	return func() tea.Msg {
+		return ToggleNodeMsg{nodeIdx: nodeIdx}
+	}
+}
+
+// Run or stop the node based on its status
+func RunNode(nodeIdx int, start bool, pid *int, args chain.MultiNodeArgs, appd string) tea.Cmd {
+	return func() tea.Msg {
+		if start {
+			nodeHome := filepath.Join(args.OutputDir, args.NodeDirPrefix+strconv.Itoa(nodeIdx))
+			// Create the command to run in background as a daemon
+			cmd := exec.Command(appd, "start", "--home", nodeHome)
+
+			// Start the process as a daemon
+			cmd.SysProcAttr = &syscall.SysProcAttr{
+				Setpgid: true, // Ensure it runs in a new process group
+			}
+
+			err := cmd.Start() // Start the node in the background
```

Comment:
The UI should look more like a dashboard. Think of a tmux with x windows displaying the log of each binary live.
The simpler UI that you have made is still great, and I still we should keep it as well, but by default we should open the more dashboard like UI.

Reply:
Thanks for your suggestion. But if you want more information about the chain, you can check the chain status via `mychaind status --node <IP-node1>`. I will add node config information so that users can easily interact with the chain.

Reply:
Right, but displaying all 4 nodes logs live is as well nice

Reply:
Hey, we've discussed internally, we still would like a UI like dashboard using bumbletea.
Maybe you should use https://github.com/charmbracelet/lipgloss?
I guess we can limit the dashboard view to 4 validators, and more fallback to the simpler view.
but this is an important point of the grant.